x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClusterUsageTelemetryIT.java

```diff
@@ -197,16 +197,15 @@ public void testAsync() throws Exception {
         assertPerClusterCount(perCluster.get(LOCAL_CLUSTER), 2L);
     }
 
+    @SuppressWarnings("checkstyle:LineLength")
```

Comment:
Can't we cut the chain in pieces here? It looks weird that we have to use this here